logging/api/Logging.Samples.Tests/DeleteSinkTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+[Collection(nameof(TestFixture))]
+public class DeleteSinkTest
+{
+    private readonly TestFixture _testFixture;
+
+    public DeleteSinkTest(TestFixture testFixture)
+    {
+        _testFixture = testFixture;
+    }
+
+    [Fact]
+    public void DeleteSink()
+    {
+        string sinkId = "sinkForDeleteSinkTest" + _testFixture.RandomName();
+        string logId = "logForDeleteSinkTest" + _testFixture.RandomName();
+        string message = "Example log entry.";
+        _testFixture.LogsToDelete.Add(logId);
+
+        // Try creating log with log entry.
+        var createLog = new WriteLogEntrySample();
+        var log = createLog.WriteLogEntry(_testFixture.ProjectID, logId, message);
+        Assert.NotNull(log);
+        Assert.Contains(message, log.TextPayload);
+
+        // Try creating sink.
+        var createSink = new CreateSinkSample();
+        var sink = createSink.CreateSink(_testFixture.ProjectID, sinkId, logId);
+        Assert.NotNull(sink);
+        Assert.Equal(sinkId, sink.Name);
+
+        // Try deleteing sink
+        var deleteSink = new DeleteSinkSample();
+        deleteSink.DeleteSink(_testFixture.ProjectID, sinkId);
+
+        Assert.Throws<Grpc.Core.RpcException>(() =>
+            this._testFixture.GetSink(this._testFixture.ProjectID, sinkId));
```

Comment:
what exception is throwing NotFound?
```suggestion
 var exception = Assert.Throws<RpcException>(() => _testFixture.GetSink(this._testFixture.ProjectID, sinkId));
Assert.Equal(StatusCode.NotFound, exception.StatusCode);
```